nucypher/cli/commands/workers.py

```diff
@@ -0,0 +1,122 @@
+"""
+ This file is part of nucypher.
+
+ nucypher is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Affero General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ nucypher is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
+"""
+
+import click
+
+try:
+    from nucypher.utilities.clouddeploy import CloudDeployers
+except ImportError:
+    CloudDeployers = None
+from nucypher.cli.utils import setup_emitter
+from nucypher.config.characters import StakeHolderConfiguration
+from nucypher.cli.commands.stake import group_staker_options, option_config_file, group_general_config
+
+
+def filter_staker_addresses(stakers, stakes):
+
+    staker_addresses = set()
+    for staker in stakers:
+        for stake in staker.stakes:
+            if stakes:
+                if not stake.staker_address in stakes:
+                    continue
+            staker_addresses.add(stake.staker_address)
+    return staker_addresses
+
+
+@click.group()
+def workers():
+    """Manage stakes and other staker-related operations."""
+
+@workers.command('create')
+@group_staker_options
+@option_config_file
+@click.option('--cloudprovider', help="currently aws", default='aws')
+@click.option('--cloud-profile', help="The cloud provider account profile you'd like to use", default=None)
+@click.option('--remote-provider', help="The blockchain provider for the remote node, if not provided nodes will run geth.", default=None)
+@click.option('--nucypher-image', help="The docker image containing the nucypher code to run on the remote nodes.", default=None)
+@click.option('--seed-network', help="Do you want the 1st node to be --lonely and act as a seed node for this network", default=False, is_flag=True)
+@click.option('--sentry-dsn', help="a sentry dsn for these workers (https://sentry.io/)", default=None)
+@click.option('--stakes', help="one or more stakers to whom we should limit worker creation", multiple=True)
+@group_general_config
+def create(general_config, staker_options, config_file, cloudprovider, cloud_profile, remote_provider, nucypher_image, seed_network, sentry_dsn, stakes):
+    """Creates workers for all stakes owned by the user for the given network."""
+
+    emitter = setup_emitter(general_config)
+
+    if not CloudDeployers:
+        emitter.echo("You need to have ansible installed to use `nucypher workers *` commands.  (pip install ansible)")
```

Comment:
```suggestion
        emitter.echo("You need to have ansible installed to use `nucypher workers *` commands. Please run 'pip install ansible'.")
```

Reply:
This voice and tone is inconsistent with the rest of the codebase. How about "Ansible is required...."